packages/usdk/packages/upstreet-agent/packages/react-agents/classes/discord-manager.ts

```diff
@@ -368,12 +372,25 @@ export class DiscordBot extends EventTarget {
         }
       });
     };
+    const _bindProfileData = () => {
+      discordBotClient.addEventListener('profileData', (e: MessageEvent) => {
+        const {
+          username,
+          avatarUrl,
+        } = e.data;
+        discordBotClient.syncBotProfileWithAgent(this.agent, {
```

Comment:
Seems weird to pass both args here... can we clarify/simplify this? For example there are two names...